tests/Core/Analyser/EventHandler/DependsOnInternalTokenTest.php

```diff
@@ -99,4 +99,26 @@ public function testInvokeEnabled(): void
             'Propagation should be stopped if the dependent is marked @deptrac-internal'
         );
     }
+
+    public function testDefaultInternalTag(): void
+    {
+        $helper = new EventHelper([], new LayerProvider([]));
+        $handler = new DependsOnInternalToken($helper, ['internal_tag' => null]);
+
+        $event = $this->makeEvent([], ['@internal' => ['']]);
+        $handler->invoke($event);
+
+        $this->assertFalse(
+            $event->isPropagationStopped(),
+            'The @internal tag should not be used per default'
```

Comment:
I thought that `@internal` was the default as per docs.

Reply:
Oh right, I changed that per our discussion, but I didn't update the docs.